### ISSUE TITLE
post about "perfect being the enemy of good" in devrel

### ISSUE THREAD

**Issue by kaycebasques** [kaycebasques/kayce.basqu.es#24]
use web font optimization as an example

async loading fonts via JS is not too hard if you're OK with flash of fallback text

but it gets complicated if you try to handle that flash perfectly

i don't think typical web users care that much about the flash

yes, this might be a case of bleeding to death by a thousand papercuts

but, in my opinion, our guidance should cover the basic usage, and be extremely clear about the drawback, but leave it at that

in other words, web developers (or devrel) seems to revel in technical complexity and this is a case of "perfect being the enemy of good"

my colleagues may disagree with this and provide some compelling arguments as to why i'm wrong, so i reserve my right to change my mind in the future, but this is my current thinking on the topic
